src/components/WizardPattern/WizardPattern.js

```diff
@@ -0,0 +1,215 @@
+    );
+  }
+
+  render() {
+    const {
+      title,
+      show,
+      onHide,
+      onExited,
+      activeStepIndex,
+      steps,
+      nextStepDisabled,
+      stepButtonsDisabled,
+      cancelButtonText,
+      backButtonText,
+      nextButtonText,
+      closeButtonText
+    } = this.props;
+    const onFirstStep = activeStepIndex === 0;
+    const onFinalStep = activeStepIndex === steps.length - 1;
+    const activeStepStr = (activeStepIndex + 1).toString();
+
+    const prevStepUnreachable = onFirstStep || this.getPrevStep(); // || ??
+    const nextStepUnreachable = nextStepDisabled; // || ??
+
+    return (
+      <Modal
+        show={show}
+        onHide={onHide}
+        onExited={onExited}
+        dialogClassName="modal-lg wizard-pf"
+      >
+        <Wizard>
+          <Modal.Header>
+            <button
+              className="close"
+              onClick={onHide}
+              aria-hidden="true"
+              aria-label="Close"
+            >
+              <Icon type="pf" name="close" />
+            </button>
+            <Modal.Title>{title}</Modal.Title>
+          </Modal.Header>
+          <Modal.Body className="wizard-pf-body clearfix">
+            <WizardPatternBody
+              {...this.props}
+              goToStep={this.goToStep}
+              nextStepDisabled={nextStepDisabled}
+              stepButtonsDisabled={stepButtonsDisabled}
+              activeStepStr={activeStepStr}
+            />
+          </Modal.Body>
+          <Modal.Footer className="wizard-pf-footer">
+            <Button bsStyle="default" className="btn-cancel" onClick={onHide}>
+              {cancelButtonText}
+            </Button>
+            <Button
+              bsStyle="default"
+              onClick={this.onBackClick}
+              disabled={prevStepUnreachable}
+            >
+              <Icon type="fa" name="angle-left" />
+              {backButtonText}
+            </Button>
+            <Button
+              bsStyle="primary"
+              onClick={onFinalStep ? onHide : this.onNextClick}
+              disabled={nextStepUnreachable}
+            >
+              {onFinalStep ? (
+                closeButtonText
+              ) : (
+                  <React.Fragment>
+                    {nextButtonText}
+                    <Icon type="fa" name="angle-right" />
+                  </React.Fragment>
+                )}
+            </Button>
+          </Modal.Footer>
+        </Wizard>
+      </Modal>
+    );
+  }
+}
+
+WizardPattern.propTypes = {
+  activeStepIndex: PropTypes.number.isRequired,
+  show: PropTypes.bool,
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.node]),
+  onHide: PropTypes.func,
+  onExited: PropTypes.func,
+  onBack: PropTypes.func,
+  onNext: PropTypes.func,
+  onStepChanged: PropTypes.func,
+  loadingTitle: PropTypes.string,
+  loadingMessage: PropTypes.string,
+  loaded: PropTypes.bool,
+  cancelButtonText: PropTypes.string,
```

Comment:
the `buttontext` convention is absent from this codebase.. that is not to say it shouldn't exist, but for brevity's sake, consider rewording this and the following three props sans 'button' 

so instead this would be `cancelText`

Reply:
Hm ok. Yeah my only reason for including it was, in v2v we use `__('this syntax')` to evaluate all our i18n strings at compile time. With these props I can maintain that full i18n compatibility. We might want to add that to more components (or find a better solution).

Reply:
`nextText` has a nice ring to it.